docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+
+**1. Install the Heroku CLI**
+
+The Heroku tools are available as a Command Line Interface (CLI). Go ahead and [install the Heroku CLI for macOS, Windows, or Linux](https://devcenter.heroku.com/articles/getting-started-with-nodejs#set-up). On macOS, you can run the command:
+
+```shell
+brew install heroku/brew/heroku
+```
+
+Once the install is complete, we can test the Heroku CLI by displaying all of the wonderful commands available to you:
+
+```shell
+heroku help
+```
+
+> 💡 If the `heroku` command is not found, you can refresh your path by opening a new terminal session/tab.
+
+**2. Log into the Heroku CLI**
+
+The Heroku CLI connects your local machine with your Heroku account. [Sign up for a free Heroku account](https://heroku.com) and then log into the Heroku CLI with the following command:
+
+```shell
+heroku login
+```
+> 💡 If you're behind a firewall, you may need to [set the proxy environment variables](https://devcenter.heroku.com/articles/using-the-cli#using-an-http-proxy) for the Heroku CLI.
+
+**3. Confirm you're logged into the Heroku CLI**
+
+Let's check that you're logged in by displaying the account currently connected to your Heroku CLI:
+
+```shell
+heroku auth:whoami
+```
+
+You're all setup with the Heroku tools! Now let's move onto the exciting step of creating an app on Heroku.
+
+---
+
+### Create an app on Heroku
+
+It’s time to [create a Heroku app](https://devcenter.heroku.com/articles/creating-apps) using the tools that you just installed. When you create an app, you can choose a unique name or have it randomly generated.
+
+> 💡 You can [rename a Heroku app](https://devcenter.heroku.com/articles/renaming-apps) at any time, but you may need to update your Git remote and app URL.
+
+**1. Create an app on Heroku**
+
+Create an app on Heroku with a unique name:
+
+```shell
+heroku create my-unique-bolt-app-name
+```
+
+or, have some fun with a random name:
+
+```shell
+heroku create
+# Creating sharp-rain-871... done, stack is heroku-18
+# https://sharp-rain-871.herokuapp.com/ | https://git.heroku.com/sharp-rain-871.git
+```
+
+After your app is created, there are a few things worth your attention:
+
+- App name is `sharp-rain-871`
+- Web address is `https://sharp-rain-871.herokuapp.com/`
+- Empty Git remote is `https://git.heroku.com/sharp-rain-871.git`
+
+**2. Confirm Heroku Git remote**
+
+The Heroku CLI automatically adds a Git remote called `heroku` to your local repository. You can list your Git remotes to confirm `heroku` exists:
+
+```shell
+git remote -v
+# heroku	https://git.heroku.com/sharp-rain-871.git (fetch)
+# heroku	https://git.heroku.com/sharp-rain-871.git (push)
+```
+
+**3. Set environment variables on Heroku**
+
+In the Getting Started guide, you [exported your signing secret and token](http://localhost:4000/bolt-js/tutorial/getting-started#tokens-and-installing-apps) as environment variables. Now we need to set the same environment variables in your Heroku app:
```

Comment:
```suggestion
In the Getting Started guide, you [exported your signing secret and token](https://slack.dev/bolt-js/tutorial/getting-started#tokens-and-installing-apps) as environment variables. Now we need to set the same environment variables in your Heroku app:
```

Reply:
This assumes people have followed the getting started guide. It may be better to say something like "Now you'll need to add your Slack app credentials to your Heroku app. If you don't know where to find your crendentials, read the getting started guide" (with the link you had). Or something that reframes it as the getting started guide coming after what the developer needs.

Reply:
Thanks for catching the `localhost` 🤦🏻 and I've updated the section with your rewording in commit 7ee5c1d. I like how you've chose to frame it - to focus on the next task rather than drawing parallels to the Getting Started guide.